spring-cloud-gcp-pubsub-stream-binder/src/main/java/com/google/cloud/spring/stream/binder/pubsub/provisioning/PubSubChannelProvisioner.java

```diff
@@ -101,7 +100,7 @@ public ConsumerDestination provisionConsumerDestination(
         subscriptionName = "anonymous." + topicShortName + "." + UUID.randomUUID();
         this.anonymousGroupSubscriptionNames.add(subscriptionName);
       }
-      ensureSubscriptionExists(subscriptionName, topicName, deadLetterPolicy, autoCreate);
+      ensureSubscriptionExists(subscriptionName, topicName, properties.getExtension(), autoCreate);
```

Comment:
`autoCreate` can be retrieved from `properties.getExtension()`.

Reply:
Done